src/composables/useInstall.ts

```diff
@@ -17,6 +17,11 @@ export const useInstall = () => {
       throw new ReCaptchaError("Cannot generate script on server side");
     }
 
+    const el = document.getElementById(RECAPTCHA_SCRIPT_ID);
+    if (el) {
```

Comment:
Why don't you `return` the function rather than delete the el element?
```suggestion
if (el) return;
```

Reply:
The event will not work:
document.getElementById(RECAPTCHA_SCRIPT_ID)!.onload

because the script is already loaded, so I am removing it.